src/chrome/content/rules/Answers.com.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="Answers">
+        <target host="answers.com/" />
+        <target host="www.answers.com/" />
+        <target host="file*.answcdn.com/" />
```

Comment:
You can't really have ```file*.answcdn.com``` as a host. Either use ```*.answcdn.com```, or explicitly define the target for each ```file*.answcdn.com``` subdomain.